lib/containers/utils.pm

```diff
@@ -151,7 +151,7 @@ sub basic_container_tests {
     die("error: missing container $container_name") unless ($output_containers =~ m/$container_name/);
 
     # Containers' state can be saved to a docker image
-    my $ret = script_run("$runtime container exec $container_name zypper -n in curl", 600);
+    my $ret = script_retry("$runtime container exec $container_name zypper -n in curl", retry => 12, delay => 15, timeout => 600, die => 0);
```

Comment:
Why 12 retries? I have quickly checked all the places where we do retries, and I think 3 times is already a good compromise.

Reply:
Yes. I'd increase the delay however then
```suggestion
    my $ret = script_retry("$runtime container exec $container_name zypper -n in curl", retry => 3, delay => 60, timeout => 600, die => 0);
```